srfi-64.html

```diff
@@ -478,11 +478,11 @@ <h3>Test-runner</h3>
 A <dfn>test-runner</dfn> is an object that runs a test-suite,
 and manages the state.  The test group path, and the sets skip and
 expected-fail specifiers are part of the test-runner.
-A test-runner will also typically accumulate statistics about executed tests,
+A test-runner will also typically accumulate statistics about executed tests.
 </p>
 <p>
 <code>(<b>test-runner?</b> <var>value</var>)</code><br />
-True iff <code><var>value</var></code> is a test-runner object.</p>
```

Comment:
"Iff" is actually correct.  It is a standard abbreviation for "if and only  if" from mathematics.

Reply:
Ah!  My non-mathematician background is revealed :-).  I'd perhaps argue it feels out of place in a programming-related specification, where `if` is understood as `iff` anyway.  I'm not sure, so I've reverted it in my latest edit.  Thanks for the instant review!